e2e/test/scenarios/dashboard-filters/temporal-unit-parameters.cy.spec.js

```diff
@@ -403,6 +407,114 @@ describe("scenarios > dashboard > temporal unit parameters", () => {
     });
   });
 
+  describe("auto-wiring", () => {
+    it("should not auto-wire to cards without breakout columns", () => {
+      cy.createDashboardWithQuestions({
+        dashboardDetails,
+        questions: [noBreakoutQuestionDetails, singleBreakoutQuestionDetails],
+      }).then(({ dashboard }) => visitDashboard(dashboard.id));
+      editDashboard();
+      addTemporalUnitParameter();
+
+      cy.log("new mapping");
+      selectDashboardFilter(getDashboardCard(1), "Created At");
+      undoToast().should("not.exist");
+
+      cy.log("new card");
+      addQuestion(noBreakoutQuestionDetails.name);
+      undoToast().should("not.exist");
+    });
+
+    it("should auto-wire to cards with breakouts on column selection", () => {
+      cy.createDashboardWithQuestions({
+        dashboardDetails,
+        questions: [
+          noBreakoutQuestionDetails,
+          singleBreakoutQuestionDetails,
+          multiBreakoutQuestionDetails,
+        ],
+      }).then(({ dashboard }) => visitDashboard(dashboard.id));
+      editDashboard();
+      addTemporalUnitParameter();
+
+      selectDashboardFilter(getDashboardCard(1), "Created At");
+      undoToast().button("Auto-connect").click();
+      saveDashboard();
+
+      filterWidget().click();
+      popover().findByText("Year").click();
+      getDashboardCard(1).findByText("Created At: Year").should("exist");
+      getDashboardCard(2).findByText("Created At: Year").should("exist");
```

Comment:
Should we take these tests one step further and assert that undoing the action will work correctly?